src/tool/subcommands/api_cmd.rs

```diff
@@ -1557,7 +1557,9 @@ fn eth_state_tests_with_tipset<DB: Blockstore>(
             tests.push(RpcTest::identity(EthGetTransactionByHash::request((tx
                 .hash
                 .clone(),))?));
-
+            tests.push(RpcTest::identity(EthGetTransactionByHashLimited::request(
```

Comment:
yeah, I guess we need a test that has `None` epoch passed, as previously mentioned by @elmattic.

Reply:
for `EthGetTransactionByHash` we use `None` 
```
get_eth_transaction_by_hash(ctx, tx_hash, None).await
```
and for `EthGetTransactionByHashLimited` we use 
```
get_eth_transaction_by_hash(ctx, tx_hash, limit).await
```